From 6ca0a69b23aa892359088daa2b49e005d127016a Mon Sep 17 00:00:00 2001
From: Yufei Gu <yufei.gu@cloudera.com>
Date: Fri, 30 Sep 2016 10:32:31 -0700
Subject: [PATCH 1901/2748] YARN-4115. Reduce loglevel of
 ContainerManagementProtocolProxy to Debug (adhoot
 via rkanter)

(cherry picked from commit b84fb41bb6ca2d69153cf5bd61f88492538ee713)
(cherry picked from commit d07eef8c871f77a96f9febe9f90be9350bb6d70c)

Change-Id: Ia7e326b1f1673f5f824447866f8567ce1a34c050
---
 .../api/impl/ContainerManagementProtocolProxy.java |   14 ++++++++++----
 1 file changed, 10 insertions(+), 4 deletions(-)

diff --git a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-client/src/main/java/org/apache/hadoop/yarn/client/api/impl/ContainerManagementProtocolProxy.java b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-client/src/main/java/org/apache/hadoop/yarn/client/api/impl/ContainerManagementProtocolProxy.java
index eaf048d..a7a4a2b 100644
--- a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-client/src/main/java/org/apache/hadoop/yarn/client/api/impl/ContainerManagementProtocolProxy.java
+++ b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-client/src/main/java/org/apache/hadoop/yarn/client/api/impl/ContainerManagementProtocolProxy.java
@@ -106,8 +106,10 @@ public synchronized ContainerManagementProtocolProxyData getProxy(
     while (proxy != null
         && !proxy.token.getIdentifier().equals(
             nmTokenCache.getToken(containerManagerBindAddr).getIdentifier())) {
-      LOG.info("Refreshing proxy as NMToken got updated for node : "
-          + containerManagerBindAddr);
+      if (LOG.isDebugEnabled()) {
+        LOG.debug("Refreshing proxy as NMToken got updated for node : "
+            + containerManagerBindAddr);
+      }
       // Token is updated. check if anyone has already tried closing it.
       if (!proxy.scheduledForClose) {
         // try closing the proxy. Here if someone is already using it
@@ -187,7 +189,9 @@ private boolean tryCloseProxy(
       ContainerManagementProtocolProxyData proxy) {
     proxy.activeCallers--;
     if (proxy.scheduledForClose && proxy.activeCallers < 0) {
-      LOG.info("Closing proxy : " + proxy.containerManagerBindAddr);
+      if (LOG.isDebugEnabled()) {
+        LOG.debug("Closing proxy : " + proxy.containerManagerBindAddr);
+      }
       cmProxy.remove(proxy.containerManagerBindAddr);
       try {
         rpc.stopProxy(proxy.getContainerManagementProtocol(), conf);
@@ -257,7 +261,9 @@ protected ContainerManagementProtocol newProxy(final YarnRPC rpc,
       
       final InetSocketAddress cmAddr =
           NetUtils.createSocketAddr(containerManagerBindAddr);
-      LOG.info("Opening proxy : " + containerManagerBindAddr);
+      if (LOG.isDebugEnabled()) {
+        LOG.debug("Opening proxy : " + containerManagerBindAddr);
+      }
       // the user in createRemoteUser in this context has to be ContainerID
       UserGroupInformation user =
           UserGroupInformation.createRemoteUser(containerId
-- 
1.7.9.5

